packages/xrpl/test/client/errors.test.ts

```diff
@@ -1,11 +1,19 @@
 import { assert } from 'chai'
 import { XrplError, NotFoundError } from 'xrpl-local'
 
-import { setupClient, teardownClient } from '../setupClient'
+import {
+  setupClient,
+  teardownClient,
+  type XrplTestContext,
+} from '../setupClient'
 
 describe('client errors', function () {
-  beforeEach(setupClient)
-  afterEach(teardownClient)
+  let testContext: XrplTestContext
```

Comment:
This looks like it doesn't need a client setup at all

Reply:
Fixed